src/shared/translations/nl.json

```diff
@@ -203,7 +211,7 @@
eriaal-is-beschikbaar-voor-ul-li-leerkrachten-aan-een-vlaamse-erkende-onderwijsinstelling-li-li-studenten-aan-een-vlaamse-lerarenopleiding-li-li-leerlingen-van-een-vlaamse-erkende-secundaire-school-li-ul-strong-hoe-krijg-je-toegang-strong-ol-li-strong-je-bent-student-aan-een-vlaamse-lerarenopleiding-strong-br-dan-krijg-je-via-je-docent-of-hogeschool-toegang-tot-onze-beeldbank-hoe-ontdek-het-op-link-to-app-path-student-teacher-deze-pagina-link-li-li-strong-je-bent-leerling-in-een-vlaamse-erkende-secundaire-school-strong-br-vraag-een-account-via-een-van-je-leerkrachten-lees-meer-over-het-archief-voor-onderwijs-voor-leerlingen-op-link-to-app-path-for-pupils-deze-pagina-link-br-wil-je-als-leerkracht-je-leerlingen-toegang-geven-alle-info-vind-je-link-to-leerlingen-toegang-versie-leerkrachten-hier-link-li-li-strong-je-bent-lesgever-in-een-vlaamse-erkende-onderwijsinstelling-strong-br-ul-li-je-hebt-een-lerarenkaart-of-stamboeknummer-maak-dan-link-to-app-path-stamboek-hier-link-je-gratis-een-account-aan-li-li-je-hebt-geen-lerarenkaart-of-stamboeknummer-of-je-vraagt-je-af-of-je-als-lesgever-zonder-nummer-in-aanmerking-komt-voor-een-account-vraag-je-toegang-aan-via-onderstaand-formulier-we-verwerken-je-aanvraag-binnen-de-vijf-werkdagen-na-ontvangst-li-ul-li-ol": "TODO: link naar FAQ BP toegang zonder lerarenkaartnummer\n <br />\n <br />\n Het Archief voor Onderwijs biedt op een eenvoudige manier toegang tot Vlaams\n audiovisueel materiaal van meer dan 30 partners. Dit materiaal is beschikbaar voor:\n <ul>\n <li>leerkrachten aan een Vlaamse erkende onderwijsinstelling</li>\n <li>studenten aan een Vlaamse lerarenopleiding</li>\n <li>leerlingen van een Vlaamse erkende secundaire school</li>\n </ul>\n <strong>Hoe krijg je toegang?</strong>\n <ol>\n <li>\n <strong>Je bent student aan een Vlaamse lerarenopleiding?</strong>\n <br />\n Dan krijg je via je docent of hogeschool toegang tot onze beeldbank. Hoe? Ontdek het\n op <Link to={APP_PATH.STUDENT_TEACHER}>deze pagina</Link>.<br />\n <br />\n </li>\n <li>\n <strong>Je bent leerling in een Vlaamse erkende secundaire school?</strong>\n <br />\n Vraag een account via een van je leerkrachten. Lees meer over Het Archief voor\n Onderwijs voor leerlingen op <Link to={APP_PATH.FOR_PUPILS}>deze pagina</Link>.\n <br />\n Wil je als leerkracht je leerlingen toegang geven? Alle info vind je{' '}\n <Link to={'/leerlingen-toegang-versie-leerkrachten'}>hier</Link>.<br />\n <br />\n </li>\n <li>\n <strong>Je bent lesgever in een Vlaamse erkende onderwijsinstelling?</strong>\n <br />\n\n <ul>\n <li>\n Je hebt een lerarenkaart- of stamboeknummer? Maak dan{' '}\n <Link to={APP_PATH.STAMBOEK}>hier</Link> je gratis een account aan.\n </li>\n\n <li>\n Je hebt geen lerarenkaart- of stamboeknummer? Of je vraagt je af of je als\n lesgever zonder nummer in aanmerking komt voor een account? Vraag je toegang aan\n via onderstaand formulier. We verwerken je aanvraag binnen de vijf werkdagen na\n ontvangst.\n </li>\n </ul>\n </li>\n </ol>",
```

Comment:
deze mogen niet vervangen worden
dit is wss een foutje in de translate extractor, ik onderzoek en doe een pr op jouw pr